docker-for-mac/release-notes.md

```diff
@@ -13,6 +13,39 @@ This page contains information about the new features, improvements, known issue
 
 {% include eula.md %}
 
+## Docker Desktop 3.6.0
+2021-08-10
+
+[Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn}
+[Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn}
+
+### New
+
+- You can now create a Dev Environment from your local git repository
+- Add sorting to volume list
+- Add name search to volumes list
```

Comment:
I'm not sure about this one. But I think it should go into the bottom section (Bug fixes and minor changes) with a prefix:

```
- Dev Environments: You can now create a Dev Environment from your local git repository
- Volume Management: Add sorting to volume list
- Volume Management: Add name search to volumes list
```